### PR TITLE
Add support in playground for relative $refs in api definition

### PR DESCRIPTION
**Pull request by ivangsa**
Playground does not support relative urls in $refs

Solution:
When loading code from url:
  it sends the api definition url as http header
  json-ref-parser will use then use this value as base file for dereferencing
Also, when loading from url param, input field is filled up for later use.

It was tested with this api definition:
https://raw.githubusercontent.com/ivangsa/playground/master/examples/multiple-files/asyncapi.yml

### CODE REVIEW

**Comment by fmvilas** [comment]
Does it still work if the AsyncAPI file is not loaded from a remote URL?

**Comment by ivangsa** [comment]
> Does it still work if the AsyncAPI file is not loaded from a remote URL?

Yes, if "Import Document" field is left blank, otherwise it will pick that url as base..

File: src/server/middlewares/version.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ module.exports = async (req, res, next) => {
 
     try {
       doc = await AsyncAPIParser.parse(req.body, {
+        path: req.header('AsyncAPI.baseUrl'),
```
Comment (fmvilas) [inline src/server/middlewares/version.js]: Minor change: could you please use a name like `x-asyncapi-base-url` for the header? Just to follow conventions on custom headers.

Comment (ivangsa) [reply src/server/middlewares/version.js]: renamed

Comment (fmvilas) [reply src/server/middlewares/version.js]: Looking great now but your code linter messed things up and changed a bunch of things. Can you please fix it back?

**Pull request merged**